AGENT_CHANGELOG.md

```diff
@@ -1,3 +1,180 @@
+* Fluentd [2.2.1](https://github.com/DataDog/integrations-core/blob/master/fluentd/CHANGELOG.md)
+* FoundationDB [1.2.1](https://github.com/DataDog/integrations-core/blob/master/foundationdb/CHANGELOG.md)
+* Gearman [2.3.1](https://github.com/DataDog/integrations-core/blob/master/gearmand/CHANGELOG.md)
+* Gitlab Runner [3.4.0](https://github.com/DataDog/integrations-core/blob/master/gitlab_runner/CHANGELOG.md)
+* Gitlab [6.2.0](https://github.com/DataDog/integrations-core/blob/master/gitlab/CHANGELOG.md)
+* GlusterFS [1.5.1](https://github.com/DataDog/integrations-core/blob/master/glusterfs/CHANGELOG.md)
+* Go Expvar [2.2.1](https://github.com/DataDog/integrations-core/blob/master/go_expvar/CHANGELOG.md)
+* Gunicorn [2.3.2](https://github.com/DataDog/integrations-core/blob/master/gunicorn/CHANGELOG.md)
+* HAProxy [4.6.1](https://github.com/DataDog/integrations-core/blob/master/haproxy/CHANGELOG.md)
+* Harbor [2.2.1](https://github.com/DataDog/integrations-core/blob/master/harbor/CHANGELOG.md)
+* Hazelcast [2.1.1](https://github.com/DataDog/integrations-core/blob/master/hazelcast/CHANGELOG.md)
+* HDFS Datanode [3.2.1](https://github.com/DataDog/integrations-core/blob/master/hdfs_datanode/CHANGELOG.md)
+* HDFS Namenode [3.2.1](https://github.com/DataDog/integrations-core/blob/master/hdfs_namenode/CHANGELOG.md)
+* Hive [1.8.2](https://github.com/DataDog/integrations-core/blob/master/hive/CHANGELOG.md)
+* HiveMQ [1.6.2](https://github.com/DataDog/integrations-core/blob/master/hivemq/CHANGELOG.md)
+* HTTP [8.3.0](https://github.com/DataDog/integrations-core/blob/master/http_check/CHANGELOG.md)
+* Hudi [2.1.2](https://github.com/DataDog/integrations-core/blob/master/hudi/CHANGELOG.md)
+* HyperV [1.9.0](https://github.com/DataDog/integrations-core/blob/master/hyperv/CHANGELOG.md)
+* IBM ACE [1.2.1](https://github.com/DataDog/integrations-core/blob/master/ibm_ace/CHANGELOG.md)
+* IBM Db2 [1.11.3](https://github.com/DataDog/integrations-core/blob/master/ibm_db2/CHANGELOG.md)
+* IBM i [1.5.1](https://github.com/DataDog/integrations-core/blob/master/ibm_i/CHANGELOG.md)
+* IBM MQ [4.2.0](https://github.com/DataDog/integrations-core/blob/master/ibm_mq/CHANGELOG.md)
+* IBM WAS [2.3.3](https://github.com/DataDog/integrations-core/blob/master/ibm_was/CHANGELOG.md)
+* Ignite [2.2.2](https://github.com/DataDog/integrations-core/blob/master/ignite/CHANGELOG.md)
+* IIS [2.19.1](https://github.com/DataDog/integrations-core/blob/master/iis/CHANGELOG.md)
+* Impala [1.1.2](https://github.com/DataDog/integrations-core/blob/master/impala/CHANGELOG.md)
+* Istio [4.4.1](https://github.com/DataDog/integrations-core/blob/master/istio/CHANGELOG.md)
+* JBoss/WildFly [2.0.2](https://github.com/DataDog/integrations-core/blob/master/jboss_wildfly/CHANGELOG.md)
+* journald [1.1.1](https://github.com/DataDog/integrations-core/blob/master/journald/CHANGELOG.md)
+* Kafka Consumer [3.1.3](https://github.com/DataDog/integrations-core/blob/master/kafka_consumer/CHANGELOG.md)
```

Comment:
@vivek-datadog @FlorentClarret I'm worried that this `3.1.3` wasn't transferred to the integration's changelog. It's a release that we made to [port these changes](https://github.com/DataDog/integrations-core/pull/15565) to 7.47. So it's [present on the release branch](https://github.com/DataDog/integrations-core/blob/03aed80d105aa81b047e74c6da086165cac5ff6f/kafka_consumer/CHANGELOG.md?plain=1#L3), but not on `master`. I guess now we know for sure that we cannot release integrations off the release branch...

Or do you think this is bug?

Reply:
I think this is fine. I would manually add a 3.1.3 changelog entry in `master` with the description of the backported changes for 7.47.

Reply:
> I guess now we know for sure that we cannot release integrations off the release branch
We should still be releasing backported changes off the release branch so that the release is compatible with the older version of the Agent. There might be some cases where new features are released in between the backported change and original release, so we don't want any unnecessary changes included in a backport release.

Reply:
Ok, I'll be sure to add this to our KB about back-porting fixes.